src/theme/mod.rs

```diff
@@ -46,7 +46,7 @@ pub struct Theme {
 }
 
 impl Theme {
-    pub fn new(src: Option<&PathBuf>) -> Self {
+    pub fn new(src: Option<&Path>) -> Self {
```

Comment:
Right, let's keep it like it is for now then. Maybe it will get refactored with all the planned changes anyway.